lib/valkyrie/persistence/fedora/list_node.rb

```diff
@@ -1,14 +1,33 @@
 # frozen_string_literal: true
 module Valkyrie::Persistence::Fedora
   # Represents a node in an ORE List. Used for persisting ordered members into
-  # an RDF Graph for Fedora, to keep order maintained.
+  # an Resource Description Framework (RDF) Graph for Fedora, to keep order maintained.
+  #
+  # RDF graph nodes are used to implement a linked list
+  # An RDF blank node is referenced for the list itself
```

Comment:
We don't use blank nodes - just hash URIs. Similar concept, but the functionality is different in Fedora.